x/wasm/client/cli/gov_tx.go

```diff
@@ -172,35 +140,121 @@ func ProposalInstantiateContractCmd() *cobra.Command {
 	return cmd
 }
 
-func ProposalMigrateContractCmd() *cobra.Command {
+func ProposalStoreAndInstantiateContractCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "migrate-contract [contract_addr_bech32] [new_code_id_int64] [json_encoded_migration_args]",
-		Short: "Submit a migrate wasm contract to a new code version proposal",
-		Args:  cobra.ExactArgs(3),
+		Use:   "store-instantiate [wasm file] [json_encoded_init_args] --label [text] --title [text] --description [text] --run-as [address] --admin [address,optional] --amount [coins,optional]",
+		Short: "Submit and instantiate a wasm contract proposal",
+		Args:  cobra.ExactArgs(1),
```

Comment:
Need to be 2 or fails
```suggestion
		Args:  cobra.ExactArgs(2),
```